tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/SanitizerDictionary.cs

```diff
@@ -32,52 +36,682 @@ public static ulong GetNextId()
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("client-request-id"),
+                    "AZSDK2017"
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("MS-CV"),
+                    "AZSDK2018"
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("X-Azure-Ref"),
+                    "AZSDK2019"
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("x-ms-request-id"),
+                    "AZSDK2020"
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("x-ms-client-request-id"),
+                    "AZSDK2021"
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("x-ms-content-sha256"),
+                    "AZSDK2022"
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("Content-Security-Policy-Report-Only"),
+                    "AZSDK2023"
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("Repeatability-First-Sent"),
+                    "AZSDK2024"
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("Repeatability-Request-ID"),
+                    "AZSDK2025"
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("repeatability-request-id"),
+                    "AZSDK2026"
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("repeatability-first-sent"),
+                    "AZSDK2027"
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("P3P"),
+                    "AZSDK2028"
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("x-ms-ests-server"),
+                    "AZSDK2029"
+                ),
+                #endregion
+                #region BodyRegex
+                new RegisteredSanitizer(
+                    new BodyRegexSanitizer(regex: "(client_id=)(?<cid>[^&\\\"]+)", groupForReplace: "cid"),
+                    "AZSDK3000"
+                ),
+                new RegisteredSanitizer(
+                    new BodyRegexSanitizer(regex: "client_secret=(?<secret>[^&\\\"]+)", groupForReplace: "secret"),
+                    "AZSDK3001"
                 ),
+                new RegisteredSanitizer(
+                    new BodyRegexSanitizer(regex: "client_assertion=(?<secret>[^&\\\"]+)", groupForReplace: "secret"),
+                    "AZSDK3002"
+                ),
+                new RegisteredSanitizer(
+                    new BodyRegexSanitizer(regex: "token=(?<token>[^&\\\"]+)", groupForReplace: "token"),
```

Comment:
```suggestion
                    new BodyRegexSanitizer(regex: "token=(?<token>[^&\\\"\\s\\n,\\\\]+)", groupForReplace: "token"),
```

I think it should be this right? To escape the newline.

Reply:
I'm taking the change, just slightly adjusting it 👍